src/core_types.jl

```diff
@@ -150,17 +150,18 @@ Base.inv(r::RotMatrix) = RotMatrix(r.mat')
 end
 
 """
-    struct Angle2d{T} <: Rotation{2,T}
-        theta::T
+    struct Angle2d{T,A} <: Rotation{2,T}
+        theta::A
     end
 
 A 2×2 rotation matrix parameterized by a 2D rotation by angle.
 Only the angle is stored inside the `Angle2d` type, values
 of `getindex` etc. are computed on the fly.
 """
-struct Angle2d{T} <: Rotation{2,T}
-    theta::T
-    Angle2d{T}(theta) where T = new{T}(theta)
+struct Angle2d{T,A} <: Rotation{2,T}
+    theta::A
+		Angle2d{T,A}(theta::Number) where{T,A} = new{T,A}(theta)
```

Comment:
```suggestion
    Angle2d{T,A}(theta::Number) where{T,A} = new{T,A}(theta)
```
We prefer four spaces instead of a tab.